host/xtest/regression_4000.c

```diff
@@ -3600,6 +3600,8 @@ static bool create_key(ADBG_Case_t *c, TEEC_Session *s,
 	return true;
 }
 
+#define XTEST_NO_CURVE 0xFFFFFFFF /* implementation-defined as per GP spec */
```

Comment:
this is a bit rude, but does the job

Reply:
Yes ;-) this is local to this file so I considered not worth it to introduce a more complex mechanism.